### PR TITLE
feat(doc): new homepage design and font

### PR DESCRIPTION
**Pull request by alextran1502**

### CODE REVIEW

File: docs/src/pages/index.tsx [hunk 1]
```diff
@@ -7,41 +7,74 @@ function HomepageHeader() {
...
+              video management{' '}
+            </span>
+            solution<span className="block"></span>
+          </p>
+
+          <p className="max-w-1/4 m-auto mt-4 px-4">
+            Easily back up, organize, and manage your photos on your own server. Immich helps you
+            <span className="sm:block"></span> browse, search and organize your photos and videos with ease, without
+            sacrifice your privacy.
           </p>
```
Comment (bo0tzz) [inline docs/src/pages/index.tsx:38]:
```suggestion
          <p className="max-w-1/4 m-auto mt-4 px-4">
            Easily back up, organize, and manage your photos on your own server. Immich helps you
            <span className="sm:block"></span> browse, search and organize your photos and videos with ease, without
            sacrificing your privacy.
          </p>
```

Maybe there should be a line break before "Immich" here.

File: docs/static/img/app-qrcode-dark.svg
Comment (bo0tzz) [inline docs/static/img/app-qrcode-dark.svg:1]: This might be a bit too pale? No idea how to fix it though.

File: docs/static/img/logomark-dark.png
Comment (bo0tzz) [inline docs/static/img/logomark-dark.png:1]: Are the colours on this different?

**Review by bo0tzz — COMMENTED** [review]
Damn shiny

File: docs/static/img/logomark-dark.png
Comment (alextran1502) [reply docs/static/img/logomark-dark.png:1]: yeah Sanjin have it faded a little bit

File: docs/src/pages/index.tsx [hunk 2]
```diff
@@ -7,41 +7,74 @@ function HomepageHeader() {
...
             Get started
           </Link>
 
           <Link
-            className="flex place-items-center place-content-center py-3 px-8 border bg-immich-primary/10 dark:bg-gray-300  rounded-full hover:no-underline text-immich-primary dark:text-immich-dark-bg font-bold uppercase"
+            className="flex place-items-center place-content-center py-3 px-8 border bg-immich-primary/10 dark:bg-gray-300  rounded-xl hover:no-underline text-immich-primary dark:text-immich-dark-bg font-bold uppercase"
             to="https://demo.immich.app/"
           >
-            Demo portal
+            Live Demo
```
Comment (danieldietzler) [inline docs/src/pages/index.tsx:53]: Maybe just `Demo`?

**Review by danieldietzler — APPROVED** [review]

File: docs/src/pages/index.tsx (hunk 1 above)
Comment (alextran1502) [reply docs/src/pages/index.tsx:38]: It is intentional for this purpose 
image

File: docs/static/img/logomark-dark.png
Comment (bo0tzz) [reply docs/static/img/logomark-dark.png:1]: Hm, not sure how I feel about it tbh.

Comment (zackpollard) [reply docs/static/img/logomark-dark.png:1]: Yea, personally not a fan of the faded colors. Our logo should be our logo, and the colors should be the same everywhere.

Comment (zackpollard) [reply docs/static/img/logomark-dark.png:1]: Yea, personally not a fan of the faded colors. Our logo should be our logo, and the colors should be the same everywhere.

Also, we shouldn't be using PNGs for this. These can easily be represented as SVGs and will be much smaller. Worst case they should be a WebP file.

**Review by zackpollard — DISMISSED** [review]

File: docs/static/img/logomark-dark.png
Comment (bo0tzz) [reply docs/static/img/logomark-dark.png:1]: We already have an svg as well, we should probably just use that with some styles right?

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**